### PR TITLE
fixes #3539 - removed legacy mysql adapter

### PR DESCRIPTION
**Pull request by lzap**
SSIA

http://koji.katello.org/koji/taskinfo?taskID=69345

TODO
- remove and block from koji
- change debian package

### CODE REVIEW

**Comment by domcleal** [comment]
Should we add a Provides: foreman-mysql to mysql2?

Please also update %changelog, just add a new line in the most recent block.

**Comment by domcleal** [comment]
Also for your todo list, the manual has to be updated.

**Comment by lzap** [comment]
Oh right plus Obsoletes, done.

LZ

## 

Later,

 Lukas "lzap" Zapletal
 irc: lzap #theforeman

File: foreman.spec [hunk 1]
```diff
@@ -258,23 +258,13 @@ Meta Package to install requirements for console support
...
-%files mysql
-%{_datadir}/%{name}/bundler.d/mysql.rb
-
 %package mysql2
 Summary: Foreman mysql2 support
 Group:  Applications/System
 Requires: %{?scl_prefix}rubygem(mysql2)
 Requires: %{name} = %{version}-%{release}
+Obsoletes: %{name}-mysql < 1.4.0
+Provides: %{name}-mysql = 1.4.0
```
Comment (domcleal) [inline foreman.spec]: Should this line be `%{version}`?

**Comment by lzap** [comment]
https://github.com/theforeman/theforeman.org/pull/125

## 

Later,

 Lukas "lzap" Zapletal
 irc: lzap #theforeman

File: foreman.spec (hunk 1 above)
Comment (domcleal) [inline foreman.spec]: ping @lzap

Comment (lzap) [inline foreman.spec]: Yes

https://fedoraproject.org/wiki/Packaging:Guidelines#Renaming.2FReplacing_Existing_Packages

Fixing. Pushed.

## 

Later,

 Lukas "lzap" Zapletal
 irc: lzap #theforeman

**Comment by domcleal** [comment]
Thanks @lzap, merged as 63b3ed2 for Foreman 1.4.0.

**Pull request closed without merging**